dockerfiles/docker-entrypoint.sh

```diff
@@ -19,25 +19,30 @@ help() {
 	exit 0
 }
 
-if ! grep docker /proc/1/cgroup -qa; then
-    help
-fi
-
 export DEFAULT_BIND_ADDRESS="0.0.0.0:8080"
 if [ -z "${BIND_ADDRESS}" ]; then
     export BIND_ADDRESS="${DEFAULT_BIND_ADDRESS}"
 fi
 
-export SEARX_VERSION=$(su searx -c 'python3 -c "import six; import searx.version; six.print_(searx.version.VERSION_STRING)"' 2>/dev/null)
-printf 'searx version %s\n\n' "${SEARX_VERSION}"
-
 export UWSGI_SETTINGS_PATH=/etc/searx/uwsgi.ini
 export SEARX_SETTINGS_PATH=/etc/searx/settings.yml
 
+# Parse special command line
+# see docs/admin/installation-docker.rst
+# display the help message without the version
+if [ "$1" = "docs" ]; then
```

Comment:
why is this called docs and not help?

Reply:
Because the output will be included into the documentation which is produce using ```make docs```.

But ```help``` makes sense.

Reply:
Updated